frontend/src/routes/Confirm/index.js

```diff
@@ -1,14 +1,15 @@
-import React from 'react'
 import { Fab } from '@material-ui/core';
+import ArrowLeftIcon from '@material-ui/icons/ArrowLeft';
 import classNames from 'classnames';
-import ArrowLeftIcon from '@material-ui/icons/ArrowLeft'
-import { handleTested } from '../../actions/handleTested';
-import { useDispatch, useSelector } from 'react-redux'
 import Wrapper from 'components/Wrapper';
-import styles from './styles.module.css'
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { sicknessStatus, testStatus } from 'routes/types';
+import { handleTested } from '../../actions/story';
+import styles from './styles.module.css';
 
 export default function Confirm(props) {
-    const isSick = useSelector(state => state.post.sick)
+    const sick = useSelector(state => state.story.sick)
```

Comment:
This is where we should change the selector if we're going with the changes suggested above, it should be `state.story.story.sick` from how it's modeled